azerothcore/azerothcore-wotlk
# fix(DB/creature_template): Improve Ulduar

Changes:

Min, Max & Modifier Damage for whole Raid.

How to test:
Apply SQL
Restart Server and delete your Cache.
Make sure to add basic tanking gear towards Raid ilevel.

Also, you can check #3552 to get more idea what you will need for testing.

Should also close #3552 

Note: You wont be able to survive some mechanics where second tank is required - Use one more account on those fights.
